### DISCUSSION TITLE
Deposit problem

### DISCUSSION THREAD

**Discussion by Shahed714** [blockscout/blockscout#10632] [General]
Please help sent 111sidra coin but he not receive my account cut 111 sidra but he not please solution my problem 
![Messenger_creation_f1cb43c0-700b-49ae-b77c-433450ca1b17](https://github.com/user-attachments/assets/64a6c0b8-5633-4618-b310-1e2e3e3f927e)
![Messenger_creation_e53d7d21-30c3-454b-8456-f81a685c98f3](https://github.com/user-attachments/assets/81232774-0b7c-4ad9-8764-028cccd79253)

**Comment by ekawibisono** [comment]
hello @Shahed714 This is blockscout github, please contact Sidra Team. Thanks

**Comment by Adyyoung44** [comment]
I withdraw my world coin and still it hasnt refect. So how can i be helped

**Reply by GimluCom** [reply]
This chat is for subjects related to the Blockscout explorer and your issue is unfortunately out of scope of what Blockscout can help with.
Please instead contact the platform or chain you used for help with this issue.

**Comment by kevo316** [comment]
i deposited some coins and they have not reflected yet

**Reply by GimluCom** [reply]
This chat is for subjects related to the Blockscout explorer and your issue is unfortunately out of scope of what Blockscout can help with.
Please instead contact the platform or chain you used for help with this issue.
